<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
    <line>
      <![CDATA[Test started running:testlogin at:16]]>
    </line>
    <line>
      <![CDATA[Test started running:testRegistraionWithValidData at:16]]>
    </line>
    <line>
      <![CDATA[<a href="/Users/bsingh5/Documents/video/selenium-project 2.17.19 PM//src//test//java//com//companyname//projectname//screenshots//09-02-2016_14-58-testRegistraionWithValidData.png"><img src="file:////Users/bsingh5/Documents/video/selenium-project 2.17.19 PM//src//test//java//com//companyname//projectname//screenshots//09-02-2016_14-58-testRegistraionWithValidData.png" alt=""height='100' width='100'/> <br />]]>
    </line>
    <line>
      <![CDATA[testRegistraionWithValidData--Test method failed
]]>
    </line>
    <line>
      <![CDATA[Test started running:testRegistraionWithInvalidData at:16]]>
    </line>
  </reporter-output>
  <suite name="Regression Suit" duration-ms="113423" started-at="2016-09-02T09:03:15Z" finished-at="2016-09-02T09:05:09Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="113423" started-at="2016-09-02T09:03:15Z" finished-at="2016-09-02T09:05:09Z">
      <class name="com.companyname.projectname.testscripts.TC002_testRegistraion">
        <test-method status="PASS" signature="setUP()[pri:0, instance:com.companyname.projectname.testscripts.TC002_testRegistraion@1d3d58a1]" name="setUP" is-config="true" duration-ms="32543" started-at="2016-09-02T14:34:04Z" finished-at="2016-09-02T14:34:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUP -->
        <test-method status="FAIL" signature="testRegistraionWithValidData()[pri:0, instance:com.companyname.projectname.testscripts.TC002_testRegistraion@1d3d58a1]" name="testRegistraionWithValidData" duration-ms="21238" started-at="2016-09-02T14:34:36Z" description="Test Registration with valid data" finished-at="2016-09-02T14:34:58Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Registration test is fail expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Registration test is fail expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.companyname.projectname.testscripts.TC002_testRegistraion.testRegistraionWithValidData(TC002_testRegistraion.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Test started running:testRegistraionWithValidData at:16]]>
            </line>
            <line>
              <![CDATA[<a href="/Users/bsingh5/Documents/video/selenium-project 2.17.19 PM//src//test//java//com//companyname//projectname//screenshots//09-02-2016_14-58-testRegistraionWithValidData.png"><img src="file:////Users/bsingh5/Documents/video/selenium-project 2.17.19 PM//src//test//java//com//companyname//projectname//screenshots//09-02-2016_14-58-testRegistraionWithValidData.png" alt=""height='100' width='100'/> <br />]]>
            </line>
          </reporter-output>
        </test-method> <!-- testRegistraionWithValidData -->
        <test-method status="PASS" signature="testRegistraionWithInvalidData()[pri:1, instance:com.companyname.projectname.testscripts.TC002_testRegistraion@1d3d58a1]" name="testRegistraionWithInvalidData" duration-ms="10948" started-at="2016-09-02T14:34:58Z" description="Test Registration with invalid data" finished-at="2016-09-02T14:35:09Z">
          <reporter-output>
            <line>
              <![CDATA[Test started running:testRegistraionWithInvalidData at:16]]>
            </line>
          </reporter-output>
        </test-method> <!-- testRegistraionWithInvalidData -->
        <test-method status="PASS" signature="quitBrowser()[pri:0, instance:com.companyname.projectname.testscripts.TC002_testRegistraion@1d3d58a1]" name="quitBrowser" is-config="true" duration-ms="83" started-at="2016-09-02T14:35:09Z" finished-at="2016-09-02T14:35:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
      </class> <!-- com.companyname.projectname.testscripts.TC002_testRegistraion -->
      <class name="com.companyname.projectname.testscripts.TC001_testLogin">
        <test-method status="PASS" signature="setUP()[pri:0, instance:com.companyname.projectname.testscripts.TC001_testLogin@4a2a4591]" name="setUP" is-config="true" duration-ms="39715" started-at="2016-09-02T14:33:15Z" finished-at="2016-09-02T14:33:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUP -->
        <test-method status="PASS" signature="testlogin()[pri:0, instance:com.companyname.projectname.testscripts.TC001_testLogin@4a2a4591]" name="testlogin" duration-ms="8536" started-at="2016-09-02T14:33:55Z" finished-at="2016-09-02T14:34:04Z">
          <reporter-output>
            <line>
              <![CDATA[Test started running:testlogin at:16]]>
            </line>
          </reporter-output>
        </test-method> <!-- testlogin -->
        <test-method status="PASS" signature="quitBrowser()[pri:0, instance:com.companyname.projectname.testscripts.TC001_testLogin@4a2a4591]" name="quitBrowser" is-config="true" duration-ms="92" started-at="2016-09-02T14:34:04Z" finished-at="2016-09-02T14:34:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quitBrowser -->
      </class> <!-- com.companyname.projectname.testscripts.TC001_testLogin -->
    </test> <!-- Test -->
  </suite> <!-- Regression Suit -->
</testng-results>
